src/documents/forms/__tests__/EditDocumentFormValidation.test.js

```diff
@@ -76,242 +76,14 @@ it('blocks submission for invalid Document Name input', async () => {
     });
```

Comment:
```
it('blocks submission for invalid Document Name input', async () => {
```
The wording could use updated to reflect that this does not block submission. 

We also need to add another test that the exact matching works.

Reply:
Similar match and exact match are tested in `rc/documents/views/__tests__/NewDocumentView.test.js` line 44 - line 63.